pennylane/ops/qubit/qchem_ops.py

```diff
@@ -535,8 +563,10 @@ class OrbitalRotation(Operation):
 
     * Number of wires: 4
     * Number of parameters: 1
-    * Gradient recipe: The ``OrbitalRotation`` operator satisfies the four-term parameter-shift rule
-      (see Appendix F, https://arxiv.org/abs/2104.05695)
+    * Gradient recipe: The ``OrbitalRotation`` operator does *not* satisfy the
+      four-term parameter-shift rule but a higher-order general parameter-shift rule
+      (with 4 equidistant frequencise :math:`\{0.5, 1, 1.5, 2\}` see,
+      https://arxiv.org/abs/2107.12390)
```

Comment:
Really nice catch! Although I might simply say
```suggestion
    * Gradient recipe: The ``OrbitalRotation`` operator has 4 equidistant frequencies
      :math:`\{0.5, 1, 1.5, 2\}`, and thus permits an 8-term parameter-shift rule.
      (see https://arxiv.org/abs/2107.12390).
```

Reply:
That is, saying it does not support a 4-term rule only really makes sense if you know we used to say it does 😆